lndmanage/lib/data_types.py

```diff
@@ -46,3 +46,18 @@ class NodeProperties:
     remote_base_fees: list
     remote_balances: list
     sent_received_per_week: int
+
+
+class KeyPair(tuple):
+    """KeyPair represents a node pair mapped to a fixed order."""
+    def __new__(cls, keys):
+        seq = (keys[0], keys[1]) if keys[0] < keys[1] else (keys[1], keys[0])
```

Comment:
Suggestion: Make it case-insensitive:
`         seq = (keys[0], keys[1]) if keys[0].casefold() < keys[1].casefold() else (keys[1], keys[0])`.


This will also make it raise when keys are not instances of (subclasses of) str, which I guess is desirable.